### DISCUSSION TITLE
posts/custom-docker-image/

### DISCUSSION THREAD

**Discussion by giscus** [techno-tim/techno-tim.github.io#245] [Comments]
# posts/custom-docker-image/

We spin up all types of containers on my channel in my tutorials, but we have yet to build our own custom Docker container image. Today we’ll start from scratch with an empty Dockerfile and create, build, and run our very own custom Docker image! We’ll learn all the commands that everyone should know when building and maintaining images with Docker. This tutorial is a great way to get started with Docker!

https://technotim.live/posts/custom-docker-image/

**Comment by Phil-Mwago** [comment]
I did not find the HTML code

**Reply by timothystewart6** [reply]
Hey!  It's in the post!  See the source files link

**Comment by nashicoco** [comment]
hi, i followed  your tutorial on docker containers but when i try accessing on my localhost with the port number, i keep getting the "Welcome to nginx" page. can you help?
